src/pages/postgraphile/make-extend-schema-plugin.md

```diff
@@ -535,6 +535,62 @@ const MyRegisterUserMutationPlugin = makeExtendSchemaPlugin(build => {
 Note that the `@pgField` directive here is necessary for PostGraphile to "look
 ahead" and determine what to request from the database.
 
+#### Working with arrays via `json_array_elements`
+
+Here's an example of working with a join table, and bulk inserting multiple records from a graphQL List.
+
+```js
+...
+
+typeDefs: gql`
+  input updatePersonsThingsInput {
+    personId: UUID!,
+    thingIds: [UUID!]!
+  }
+  extend type Mutation {
+    updatePersonsThings(input: updatePersonsThingsInput!): PersonsThingsConnection
```

Comment:
We always follow the input object, payload object mutation pattern (see https://www.apollographql.com/blog/graphql/basics/designing-graphql-mutations/ ); and there's no point having a connection on a mutation payload since you cannot paginate over it (without performing the mutation again, or using two different operations), so we only use lists on mutation payloads:

```suggestion
  type UpdatePersonThingsPayload {
    personThings: [PersonThings!]
  }
  extend type Mutation {
    updatePersonsThings(input: UpdatePersonsThingsInput!): UpdatePersonsThingsPayload
```